depend/zcash/src/zcash/cache.cpp

```diff
@@ -3,16 +3,16 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include "zcash/cache.h"
-#include "util/system.h"
+// #include "util/system.h"
```

Comment:
I think we can leave this as it is as they are the effects of applying the patch. We can modify the patch to delete instead of commenting out but it seems unnecessary.